Octokit/Clients/IRepositoryContentsClient.cs

```diff
@@ -24,6 +25,22 @@ public interface IRepositoryContentsClient
         Task<IReadOnlyList<RepositoryContent>> GetAllContents(string owner, string name, string path);
 
         /// <summary>
+        /// Returns the contents of a file or directory in a repository.
+        /// </summary>
+        /// <remarks>
+        /// If given a path to a single file, this method returns a collection containing only that file.
+        /// See the <a href="https://developer.github.com/v3/repos/contents/#get-contents">API documentation</a> for more information.
+        /// </remarks>
+        /// <param name="owner">The owner of the repository</param>
+        /// <param name="name">The name of the repository</param>
+        /// <param name="path">The content path</param>
+        /// <param name="reference">The name of the commit/branch/tag. Default: the repository�s default branch (usually master)</param>
+        /// <returns>
+        /// A collection of <see cref="RepositoryContent"/> representing the content at the specified path
+        /// </returns>
+        Task<IReadOnlyList<RepositoryContent>> GetContents(string owner, string name, string path, string reference);
```

Comment:
In #771 we added a rule to the test suite to ensure all methods which returned a `IReadOnlyList<T>` were named with the prefix `GetAll` - if you update this name and the related observable method name you should be :sparkles: